client/src/components/Lesson/WordCreator/ContextSentences.tsx

```diff
@@ -26,6 +29,10 @@ export class ContextSentences extends React.Component<ContextSentencesProps, Con
 
     constructor(props: ContextSentencesProps) {
         super(props);
+        this.state = {
+            value: ([] as string[]),
+            error: ([] as string[]),
```

Comment:
that form of defining an empty string array doesn't work, this works, if i knew what the issue was with this I could respond better.